### DISCUSSION TITLE
Requirements for Bowflex C6 / Schwinn IC4 / IC8

### DISCUSSION THREAD

**Discussion by chinasaur** [doudar/SmartSpin2k#209] [Q&A] [answered]
I switched from a Saris Hammer to Bowflex C6 so my partner can use it too (also so much quieter). I miss the automatic resistance control though. Really happy to find this project; I was thinking about putting a stepper on this somehow and I have some ESP32 boards around.

Since I'm only planning to use this specific bike, I wonder about the hardware requirements. The knob doesn't need much torque, but needs 8-9 full revolutions. Anyone have gearing suggestions? And I was considering using a smaller stepper too.

Thanks!

**Comment by doudar** [comment, marked as answer]
Should work great with the C6 - it’s the same as the IC4. You can probably get away with a smaller stepper on that bike as it doesn’t seem to require much torque and if it doesn’t work, changing out to a bigger stepper is pretty easy.

Keep us updated and post some build pictures if you can. Have fun!

**Reply by chinasaur** [reply]
Thanks! Do you think just your standard gearing will work well for this bike? I don't know if the C6 resistance change per knob degree is typical. And I don't have a great sense how fast the resistance changes need to happen in order to have a good ride feel.

**Reply by doudar** [reply]
Yep, the standard 11/30 gear combo works well with it.
